rcode-reader/src/main/java/eu/rssw/pct/elements/DataRelationElement.java

```diff
@@ -0,0 +1,52 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import eu.rssw.pct.RCodeInfo;
+
+public class DataRelationElement extends ClassElement {
+  private int pairCount;
+  private int flags;
+
+  private String parentBufferName;
+  private String childBufferName;
+  private String fieldPairs;
+
+  public DataRelationElement(byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
+    this.pairCount = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
+    this.flags = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(order).getShort();
+
+    int parentBufferNameOffset = ByteBuffer.wrap(segment, currentPos + 8, Integer.BYTES).order(order).getInt();
+    this.parentBufferName = parentBufferNameOffset == 0 ? ""
+        : RCodeInfo.readNullTerminatedString(segment, textAreaOffset + parentBufferNameOffset);
+
+    int child_buffer_name_offset = ByteBuffer.wrap(segment, currentPos + 12, Integer.BYTES).order(order).getInt();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00117)